dowhy/utils/graph_operations.py

```diff
@@ -138,3 +139,18 @@ def dfs(node_idx, component):
             c_components.append(component)
 
     return c_components
+
+def daggity_to_dot(daggity_string):
+    '''
+    Converts the input daggity_string to valid DOT graph format.
+
+    :param daggity_string: Output graph from Daggity site
+    :returns: DOT string  
+    '''
+    graph = re.sub(r'\n' , '; ' , daggity_string)
+    graph = re.sub(r'^dag ', 'digraph ', graph)
+    graph = re.sub('{;' , '{' , graph)
+    graph = re.sub('};' , '}' , graph)
+    graph = re.sub('[a-zA-Z]*,' , '' , graph)
```

Comment:
is this line removing all the parameters inside `[ ]`? `pos` actually is a valid dot attribute that helps in drawing the graph. So we don't want to remove that. 
Also, if you have an unobserved node, then dagitty labels it as "latent". Instead, dowhy requires users to specify `observed=no` inside []. 
So there is some more work here. 

1. exposure and outcome can be removed.
2. latent needs to be replaced with observed=no
3. pos can stay as it is.

Reply:
No this doesn't remove pos parameter. I will add the latent part in next commit.